packages/@aws-cdk/aws-glue/lib/storage-type.ts

```diff
@@ -0,0 +1,77 @@
+/**
+ * Absolute class name of the Hadoop `InputFormat` to use when reading table files.
+ */
+export enum InputFormat {
```

Comment:
I think jsii has some issues with enums that have a single entry. Also, In order not to block people in case new types will be introduced, maybe you can use an enum-like class:

```ts
export class InputFormat {
  public static TextInputFormat = new InputFormat('org.apache.hadoop.mapred.TextInputFormat');

  constructor(public readonly format: string) { }
}
```

Reply:
+1

Reply:
Was fixed by converting the enums into enum-like classes - i also prefer the extensible approach.